dali/operators/geometry/affine_transforms/transform_crop.cc

```diff
@@ -126,9 +126,9 @@ class TransformCropCPU
     from_end_.Read(spec, ws, repeat);
     to_start_.Read(spec, ws, repeat);
     to_end_.Read(spec, ws, repeat);
-    auto sizes = {from_start_[0].size(), from_end_[0].size(),
-                  to_start_[0].size(), to_end_[0].size()};
-    ndim_ = std::max(sizes);
+    auto sizes = std::array<size_t, 4>{from_start_[0].size(), from_end_[0].size(),
```

Comment:
What was so terribly wrong with the initializer_list?

Reply:
The underlying storage has lifetime of temporary object, so it shouldn't be used to store values and pass them around this way.

Reply:
OK, whatever.